internal/cmd/agent/trigger/watcher.go

```diff
@@ -125,10 +132,40 @@ func (t *Trigger) OnChange(key string, defaultNamespace string, trigger func(),
 	return nil
 }
 
-func (t *Trigger) call(gvk schema.GroupVersionKind, key objectset.ObjectKey) {
+func (t *Trigger) storeObjectGeneration(uid types.UID, generation int64) *atomic.Int64 {
+	value := new(atomic.Int64)
+	value.Store(generation)
+	t.seenGenerations.Store(uid, value)
+	return value
+}
```

Comment:
I think this would make the linter happy:
```suggestion
func (t *Trigger) storeObjectGeneration(uid types.UID, generation int64) {
	value := new(atomic.Int64)
	value.Store(generation)
	t.seenGenerations.Store(uid, value)
}
```